ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh

```diff
@@ -144,31 +144,33 @@ steadystate() {
     echo "last:$last current:$current Waiting to reach $STEADY_STATE_MINIMUM before checking for steady-state"
     return 1
   fi
-  within $last $current $STEADY_STATE_THRESHOLD
+  within "$last" "$current" "$STEADY_STATE_THRESHOLD"
 }
 
 waitForReady() {
   logger "Recovery: Waiting ${MAXIMUM_WAIT_TIME}s for the initialization to complete"
-  local lastSystemdCpuset="$(currentAffinity 1)"
-  local lastDesiredCpuset="$(unrestrictedCpuset)"
+  local lastSystemdCpuset lastDesiredCpuset
+  lastSystemdCpuset=$(currentAffinity 1)
+  lastDesiredCpuset=$(unrestrictedCpuset)
   local t=0 s=10
   local lastCcount=0 ccount=0 steadyStateTime=0
   while [[ $t -lt $MAXIMUM_WAIT_TIME ]]; do
     sleep $s
     ((t += s))
```

Comment:
Just an fyi... bashate is also flagging some potential issues. Mostly indentation and line length (I ignore the line lengths when I add bashate and shellcheck as linters in repos), but has flagged this arithmetic expression style in a few spots:

accelerated-container-startup.sh:80:1: E043 Arithmetic compound has inconsistent return semantics
accelerated-container-startup.sh:82:1: E043 Arithmetic compound has inconsistent return semantics
accelerated-container-startup.sh:159:1: E043 Arithmetic compound has inconsistent return semantics
accelerated-container-startup.sh:174:1: E043 Arithmetic compound has inconsistent return semantics

Reply:
I don't know bashate - Sounds like something I wouldn't hate.